packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -24,6 +24,14 @@ const resources = {
   bytecodes: {
     createIndexes: [
     ]
+  },
+  networks: {
+    createIndexes: [
+    ]
+  },
+  contractInstances: {
+    createIndexes: [
+    ]
```

Comment:
This will likely need an index on `network.id`

Reply:
For `fork`, right? I'll add it. But do you mean `network.id`? Or `network.networkId`?

Reply:
Yep, `network.id`! We will want to be find contract instances by a specific network object.